website/content/docs/concepts/host-discovery/index.mdx

```diff
@@ -10,7 +10,11 @@ description: |-
 Traditionally, connecting to remote hosts and services requires knowledge of
 the endpoint’s connection info (e.g. the IP address and port of the service).
 This creates complexity when managing the onboarding of new resources at scale
-or dealing with dynamic services whose connection info frequently changes.
+or dealing with dynamic, ephemeral services whose connection info frequently changes.
+Furthermore, the increased operational overhead of having to manually manage and update
+new or old resources is an inefficient use of time.
+Resources should be tagged appropriately and depending on the identity of the end-user,
+automatically have the resources that they are allowed to connect to.
```

Comment:
```suggestion
Resources should be tagged appropriately so that, depending on their identity, users
automatically have the resources that they are allowed to connect to.
```